lib/sql_footprint/sql_anonymizer.rb

```diff
@@ -0,0 +1,16 @@
+module SqlFootprint
+  class SqlAnonymizer
+    GSUBS = {
+      /\sIN\s\((.*)\)/ => ' IN (values-redacted)'.freeze, # IN clauses
+      /\s\=\s([0-9]+)/ => ' = number-redacted'.freeze, # numbers
+      /\s'(.*)\'/ => " 'value-redacted'".freeze, # literal strings
+    }.freeze
+
+    def anonymize sql
+      # subs => [key, value]
+      GSUBS.reduce(sql) do |s, subs|
```

Comment:
Rather than `first` and `last` on the next line, use this style: `GSUBS.reduce(sql) do |s, (regex, replacement)|`